master/requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+twisted == 17.5.0
+Jinja2 == 2.8
+sqlalchemy >= 0.6, <= 0.7.9
+sqlalchemy-migrate == 0.7.2
+python-dateutil == 1.5
+autobahn == 0.11.0
+python-ldap
+mysql-python == 1.2.5
+pymysql == 0.7.1
+PyJWT == 1.4.0
+psutil == 4.3.0
```

Comment:
Aren't any of these declared in `setup.py`? IMO `requirements-dev` should only contain things that are required for development but not every day use (the every day stuff should be in setup.py)